explorer-backend/fly/migration/migration.go

```diff
@@ -136,5 +141,28 @@ func Run(db *mongo.Database) error {
 		return err
 	}
 
+	tokenTransferIndexes := []mongo.IndexModel{
+		{
+			Keys: bson.D{{Key: "tokenAddress", Value: 1}},
+		},
+		{
```

Comment:
Thanks for the information! But the doc says that in this case, it won't be as efficient as standalone indexing:

```
MongoDB can also use the index to support a query on the item and stock fields, since the item field corresponds to a prefix. However, in this case the index would not be as efficient in supporting the query as it would be if the index were on only item and stock.
```